conformance/mesh/manifests.yaml

```diff
@@ -132,3 +131,33 @@ spec:
     port: 9090
   - name: grpc
     port: 7070
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: gateway-conformance-mesh-consumer
```

Comment:
namespace needs to be added here: https://github.com/istio/gateway-api/blob/a258900338d925af89116ef29d613bc6e84be032/conformance/utils/suite/suite.go#L175 (and maybe fix the log to say "Mesh" instead of "base" while we are there), so it will wait for the pod to be ready